Sabetskyi_Stanislav/src/Class_Class.java

```diff
@@ -0,0 +1,24 @@
+import java.util.Scanner;
+
+public class Class_Class {
```

Comment:
You used C# style naming. In Java you should use camel case for naming (ClassClass with no underline)